### ISSUE TITLE
tests/wpt/webgl/tests/conformance/extensions/oes-texture-float-with-canvas.html crashes on its own

### ISSUE THREAD

**Issue by nox** [servo/servo#22113]
This test fails locally when ran without `--no-pause`.

```
pid:21839 ERROR 2018-11-05T11:47:32Z: canvas::webgl_thread: Last GL operation failed: TexImage2D(3553, 0, 34842, 1, 2, 6408, 36193, IpcBytesReceiver { os_receiver: OsIpcReceiver { port: Cell { value: 69135 } } })
```

**Comment by nox** [comment]
That may be due to the fact that `HALF_FLOAT_OES` is not equal to `HALF_FLOAT_ARB`.

**Comment by nox** [comment]
Yes, that was it, I drive-by fix this in #22112.
